domain/consensus/factory.go

```diff
@@ -126,6 +127,7 @@ func (f *factory) NewConsensus(dagParams *dagconfig.Params, db infrastructuredat
 	headersSelectedTipStore := headersselectedtipstore.New()
 	finalityStore := finalitystore.New(200, preallocateCaches)
 	headersSelectedChainStore := headersselectedchainstore.New(pruningWindowSizeForCaches, preallocateCaches)
+	daaBlocksStore := daablocksstore.New(pruningWindowSizeForCaches, preallocateCaches)
```

Comment:
Maybe split the cache size to 2:
1. DAA score at size: pruningWindowSizeForCaches.
2. Derge Set for FinalityWindow only.